build/scraper.mjs

```diff
@@ -1,17 +1,18 @@
-import lzma from 'lzma';
 import { load } from 'cheerio';
+import lzma from 'lzma';
 
 import { Generator as RelicGenerator } from '@wfcd/relics';
 import patchlogs from 'warframe-patchlogs';
 
 import Progress from './progress.mjs';
+import readJson from './readJson.mjs';
 import ModScraper from './wikia/scrapers/ModScraper.mjs';
-import WeaponScraper from './wikia/scrapers/WeaponScraper.mjs';
-import WarframeScraper from './wikia/scrapers/WarframeScraper.mjs';
 import VersionScraper from './wikia/scrapers/VersionScraper.mjs';
-import readJson from './readJson.mjs';
+import WarframeScraper from './wikia/scrapers/WarframeScraper.mjs';
+import WeaponScraper from './wikia/scrapers/WeaponScraper.mjs';
 
 import { get, getJSON, retryAttempts } from './network.mjs';
+import ArchwingScraper from './wikia/scrapers/ArchwingScaper.mjs';
```

Comment:
could you revert these changes? It seems that your formatter thought it would be better to rearrenge them 😂